data/json/items/magazine/66mm.json

```diff
@@ -3,7 +3,7 @@
     "id": "m74_clip",
     "looks_like": "belt40mm",
     "type": "MAGAZINE",
-    "name": { "str": "M74 rocket clip" },
+    "name": { "str": "M74 4 rockets clip" },
```

Comment:
```suggestion
    "name": { "str": "M74 4-rocket clip" },
```

Reply:
I would personally commit to the second one, as it seems more consistent with the naming style. It also seems to be more grammatically correct, or at the very least the first one doesn't sound right.